app/code/Magento/CmsGraphQl/Model/Resolver/Blocks.php

```diff
@@ -78,7 +78,10 @@ private function getBlocksData(array $blockIdentifiers): array
         $blocksData = [];
         try {
             foreach ($blockIdentifiers as $blockIdentifier) {
-                $blocksData[$blockIdentifier] = $this->blockDataProvider->getData($blockIdentifier);
+                $blockData = $this->blockDataProvider->getData($blockIdentifier);
+                if (!empty($blockData)) {
+                    $blocksData[$blockIdentifier] = $blockData;
+                }
```

Comment:
We need to log the fact that some of the requested blocks were not found, please add this to the `else`.

Reply:
I provided a little refactoring of this PR
Now blocks data is located in `data` section, and errors are located in `errors` section

Also, I removed the logging, because there it’s way to log overflow logfile if I send non-existent block id
And at the same time for the client, there is no information what is the actual error (error is only on the server)